starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,63 @@
+const SpotifyWebApi   = require('spotify-web-api-node');
+const spotify = new SpotifyWebApi({
+    clientId : 'b5b7561cbcb04c869bae40eed5c71269',
+    clientSecret : '990748f563da496dadb97356429c5096',
+  });
+const express         = require('express');
+const app             = express();
+const bodyParser      = require('body-parser');
+const expressLayouts  = require('express-ejs-layouts');
+app.use(express.static('public'));
+app.use(expressLayouts);
+app.use(bodyParser.urlencoded({ extended: true }));
+app.set('views', __dirname + '/views');
+app.set('layout', 'layouts/main-layout');
+app.set('view engine', 'ejs');
+
+spotify.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.